src/components/video/VideoPlayer.jsx

```diff
@@ -1,8 +1,11 @@
-import React, { useMemo } from 'react';
+import React, { lazy, Suspense, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import DelayedFallbackContainer from '../DelayedFallback/DelayedFallbackContainer';
 
-import VideoJS from './VideoJS';
-
+const VideoJS = lazy(() => import(
+  /* webpackChunkName: "videojs" */
```

Comment:
[suggestion, unrelated] FYI, i think you might be able to add `webpackChunkName` inline comment to the dynamic import of the `MyCareer` component?